src/graphics/ScreenFonts.h

```diff
@@ -16,49 +16,53 @@
 #include "graphics/fonts/OLEDDisplayFontsCS.h"
 #endif
 
-#if (defined(USE_EINK) || defined(ILI9341_DRIVER) || defined(ILI9342_DRIVER) || defined(ST7701_CS) || defined(ST7735_CS) ||      \
-     defined(ST7789_CS) || defined(USE_ST7789) || defined(HX8357_CS)) &&                                                         \
-    !defined(DISPLAY_FORCE_SMALL_FONTS)
-// The screen is bigger so use bigger fonts
-#define FONT_SMALL ArialMT_Plain_16  // Height: 19
-#define FONT_MEDIUM ArialMT_Plain_24 // Height: 28
-#define FONT_LARGE ArialMT_Plain_24  // Height: 28
-#else
 #ifdef OLED_PL
-#define FONT_SMALL ArialMT_Plain_10_PL
+#define FONT_SMALL_LOCAL ArialMT_Plain_10_PL
 #else
 #ifdef OLED_RU
-#define FONT_SMALL ArialMT_Plain_10_RU
+#define FONT_SMALL_LOCAL ArialMT_Plain_10_RU
 #else
 #ifdef OLED_UA
-#define FONT_SMALL ArialMT_Plain_10_UA // Height: 13
+#define FONT_SMALL_LOCAL ArialMT_Plain_10_UA // Height: 13
 #else
 #ifdef OLED_CS
-#define FONT_SMALL ArialMT_Plain_10_CS
+#define FONT_SMALL_LOCAL ArialMT_Plain_10_CS
 #else
-#define FONT_SMALL ArialMT_Plain_10 // Height: 13
+#define FONT_SMALL_LOCAL ArialMT_Plain_10 // Height: 13
 #endif
 #endif
 #endif
 #endif
 #ifdef OLED_UA
-#define FONT_MEDIUM ArialMT_Plain_16_UA // Height: 19
+#define FONT_MEDIUM_LOCAL ArialMT_Plain_16_UA // Height: 19
 #else
 #ifdef OLED_CS
-#define FONT_MEDIUM ArialMT_Plain_16_CS
+#define FONT_MEDIUM_LOCAL ArialMT_Plain_16_CS
 #else
-#define FONT_MEDIUM ArialMT_Plain_16 // Height: 19
+#define FONT_MEDIUM_LOCAL ArialMT_Plain_16 // Height: 19
 #endif
 #endif
 #ifdef OLED_UA
-#define FONT_LARGE ArialMT_Plain_24_UA // Height: 28
+#define FONT_LARGE_LOCAL ArialMT_Plain_24_UA // Height: 28
 #else
 #ifdef OLED_CS
-#define FONT_LARGE ArialMT_Plain_24_CS // Height: 28
+#define FONT_LARGE_LOCAL ArialMT_Plain_24_CS // Height: 28
 #else
-#define FONT_LARGE ArialMT_Plain_24 // Height: 28
+#define FONT_LARGE_LOCAL ArialMT_Plain_24 // Height: 28
 #endif
 #endif
+
+#if (defined(USE_EINK) || defined(ILI9341_DRIVER) || defined(ILI9342_DRIVER) || defined(ST7701_CS) || defined(ST7735_CS) ||      \
+     defined(ST7789_CS) || defined(USE_ST7789) || defined(HX8357_CS)) &&                                                         \
+    !defined(DISPLAY_FORCE_SMALL_FONTS)
+// The screen is bigger so use bigger fonts
+#define FONT_SMALL FONT_MEDIUM_LOCAL // Height: 19
```

Comment:
What happens here if none of the local fonts are defined?

Reply:
Font all font sizes there is always fallback to default implementation like here: https://github.com/kyberpunk/firmware/blob/fix-localization-on-big-screens/src/graphics/ScreenFonts.h#L42.